extensions/cornerstone/src/utils/getCornerstoneBlendMode.ts

```diff
@@ -11,5 +13,13 @@ export default function getCornerstoneBlendMode(blendMode: string): Enums.BlendM
     return Enums.BlendModes.MAXIMUM_INTENSITY_BLEND;
   }
 
-  throw new Error();
+  if (blendMode.toLowerCase() === MINIP) {
+    return Enums.BlendModes.MINIMUM_INTENSITY_BLEND;
+  }
+
+  if (blendMode.toLowerCase() === AVG) {
+    return Enums.BlendModes.AVERAGE_INTENSITY_BLEND;
+  }
+
+  throw new Error(`Unsupported blend mode: ${blendMode}`);
```

Comment:
This presents a better image for the user when this error happens.

<img width="1183" alt="Screenshot 2024-04-21 at 5 01 53 PM" src="https://github.com/OHIF/Viewers/assets/234049/6a2d8db2-49cf-483b-946f-a37c23721e79">